agent/vendor/github.com/aws/amazon-ecs-agent/ecs-agent/manageddaemon/managed_daemon_windows.go

```diff
@@ -0,0 +1,106 @@
+//go:build windows
+// +build windows
+
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//      http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package manageddaemon
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+)
+
+const (
+	imageTarPath                          = "C:\\ProgramData\\Amazon\\ECS\\data\\"
```

Comment:
Why do we need a separate file with the same method.
Can we just convert the values as constants declared in platform files and then keep the method common?